how-it-works/1_protocol/01_peer-to-peer-p2p.subpage.md

```diff
@@ -36,28 +36,20 @@ After receiving an advert, a node may request the corresponding artifact from on
 ## Prioritization of Artifacts
 
 The P2P layer allows the prioritization of artifacts such that the more crucial artifacts are broadcast throughout the subnet nodes more quickly than the others.
-Prioritizing some artifacts over others is important to ensure that the protocol can always make progress and not be starved of network bandwidth for "less important" traffic.
+Prioritizing some artifacts over others is important to ensure that the protocol can always make progress and not be starved of network bandwidth by "less important" traffic.
 This principle is well known from traditional networking and applies equally well to a blockchain system.
 
-## Implementation Architecture
+## Security
+To prevent Denial of Service (DOS) attacks, nodes will only request and accept connections with nodes in
+the same subnet. Subnet membership is managed by the [Network Nervous System (NNS)](https://internetcomputer.org/how-it-works/#Network-Nervous-System). 
```

Comment:
```suggestion
the same subnet. Subnet membership is managed by the [Network Nervous System (NNS)](/how-it-works/#Network-Nervous-System). 
```
Please use relative links. This way, CI can catch it if they break

Reply:
thanks, Severin!